dlt/common/configuration/specs/run_configuration.py

```diff
@@ -18,7 +18,14 @@ class RunConfiguration(BaseConfiguration):
     dlthub_telemetry_segment_write_key: str = "a1F2gc6cNYw2plyAt02sZouZcsRjG7TD"
     log_format: str = '{asctime}|[{levelname:<21}]|{process}|{name}|{filename}|{funcName}:{lineno}|{message}'
     log_level: str = "WARNING"
-    request_timeout: Tuple[int, int] = (15, 300)  # default request timeout for all http clients
+    request_timeout: float = 60
```

Comment:
Changed the type to float since union isn't supported in config, and thinking more common to use one number than the "connect, read" tuple.